### ISSUE TITLE
npm WARN npm npm does not support Node.js v8.9.4

### ISSUE THREAD

**Issue by jreviews** [ionic-team/vscode-extension#27]
I am able to run everything manually on mac os terminal and vscode terminal, but I get this error when using the ionic extension.

Part of the log below:

```
0 warn npm npm does not support Node.js v8.9.4
1 warn npm You should probably upgrade to a newer version of node as we
2 warn npm can't make any promises that npm will work with this version.
3 warn npm You can find the latest version at https://nodejs.org/
4 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'run', 'serve' ]
5 info using npm@8.1.4
6 info using node@v8.9.4
```

When running the following command in vscode terminal

```
which node
```

The path given is `/Users/me/.nvm/versions/node/v14.17.0/bin/node` which is different than what the extension is using. `/usr/local/bin/node` seems to be in fact an older version, but not sure how to get that updated.

Any ideas to get this working?

**Comment by dtarnawsky** [comment]
Hey @jreviews, Can you try setting the default node in the command prompt:
`nvm alias default 14.17.0`

Alternatively, you can install node from https://nodejs.dev/. nvm seems to keep the default node version as the first one you installed ever.

**Comment by jreviews** [comment]
Thanks @dtarnawsky 

I had tried the first option before without success. I ended up renaming the system node file and replacing it with a symlink to the nvm path and that seems to work.

It'd be great if the extension would automatically check the "which node" path and use the output to execute the npm commands rather than the system default. Or at least have an option for setting the node path in the extension settings.

Looking forward to working with the extension!
